website/docs/sdk-reference/ios.md

```diff
@@ -116,22 +134,55 @@ client.getValue(
         doTheOldThing()
     }
 }
+
+// Or with async/await
+let isMyAwesomeFeatureEnabled = await client.getValue(
+    for: "keyOfMySetting", // Setting Key
+    defaultValue: false, // Default value
+    user: ConfigCatUser(identifier: "435170f4-8a8b-4b67-a723-505ac7cdea92") // Optional User Object
+
+if isMyAwesomeFeatureEnabled {
+    doTheNewThing()
+} else {
+    doTheOldThing()
+}
 ```
 
-## Anatomy of asynchronous `getValue()`
+## Anatomy of `getValueDetails()`
```

Comment:
Pleas add a sentence about what is this the purpose of this function

Reply:
done